docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -7,17 +7,46 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-This document describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC and Mutual TLS authentication as well as the proactive authentication.
+The following section describes the Quarkus built-in authentication mechanisms for HTTP-based BASIC, FORM, and both Mutual TLS and proactive authentication.
 
 [[basic-auth]]
 == Basic Authentication
+ 
+HTTP Basic authentication (BA) uses fields in the HTTP header, and is the easiest to set up and least resource demanding technique for enforcing access controls to Web resources without requiring HTTP cookies, session identifiers, or login pages.
```

Comment:
I'm not sure having a `BA` acronym is useful, I'd rather remove it

Reply:
Yeah, tbh, I had the same comment and refrained from doing it. I have never seen it in the wild (I just saw it on the Wikipedia page while checking if it was a thing).